index.d.ts

```diff
@@ -53,11 +53,14 @@ export interface FieldProps {
 export class Field extends React.Component<FieldProps, any> {
 }
 
+export type OnSubmitCallback = (s: {name: string, values: {[key: string]: any}}) => void;
```

Comment:
You can imagine it as `type object = {[key: string]: any}}`, therefore yes, it's better 'cause it's former and shorter.